javascript.md

```diff
@@ -303,12 +303,12 @@ for (let i = 0; i < array.length; i++) {
 	newArray.push(array[i] * 3);
 }
 ```
+## <a name="errors">Error Handling</a>
 
-## <a name="json">JSON</a>
+### 8.1 Displaying Errors
+> Errors happen. If an error blocks or inhibits the user flow, we should always display the error to the user. You can `console.error()` errors if useful for developers, but do not rely on this for end users.
```

Comment:
Should we mention the `debug()` function in Ember over `console.error()`? Note that lint will complain about any console logging statements.

Reply:
I agree with @ASH-Bryan. I think that the use of the `debug(..)` method should be used instead of `console.error(..)`, especially since it gets stripped out in production.

Reply:
i'll make this update